docs/DEPLOYMENT.md

```diff
@@ -1,5 +1,19 @@
 # Deployment
 
+- [Prerequisites](#prerequisites)
+- [Deploy API Service](#deploy-api-service)
+  - [Update API Secret](#update-the-api-secret)
+  - [Update API Config Map](#update-api-config-map)
+  - [Prepare API and Db Migration release yaml](#Prepare-the-api-and-db-migration-release-yaml)
+  - [Setup Ingress/Route](#setup-route/ingress)
+- [Deploy UI](#deploy-ui)
+  - [Build and Publish Image](#build-and-publish-image)
+  - [Update the UI ConfigMap](#update-the-ui-configmap)
+  - [Setup Ingress/Route](#setup-route/ingress)
+- [Add resources in DB](#add-resources-in-db)
+- [Setup Catalog Refresh CronJob](#setup-catalog-refresh-cronjob)
+- [Adding New Users in Config](#adding-new-users-in-config)
```

Comment:
Are we missing to add a heading and description for adding a jwt token which does not gets expired ??

Reply:
that is in the catalog refresh part